Makefile.common

```diff
@@ -3,14 +3,25 @@ VCS_REF       := $(shell git rev-parse HEAD)
 VCS_URL       := https://github.com/elastic/golang-crossbuild
 BUILD_DATE    := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 .DEFAULT_GOAL := build
+SUFFIX_NPCAP_VERSION := -npcap-$(NPCAP_VERSION)
+NPCAP_REPOSITORY := docker.elastic.co/observability-ci
 
-# Requires login at https://docker.elastic.co:7000/.
 push:
-	echo ">> Pushing $(REPOSITORY)/$(NAME):$(VERSION)$(SUFFIX)$(TAG_EXTENSION)"
+	@[ "$(SUFFIX)" = "$(SUFFIX_NPCAP_VERSION)" ] \
```

Comment:
I could not find the way to use `ifeq` with SUFFIX and SUFFIX_NPCAP_VERSION, even though these values were the same, but the `ifeq` returned false.

I guess, something should be related to where the SUFFIX is defined and the `include` function.